cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -703,11 +712,13 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
             value: eventHubNamespace.outputs.name
           }
           {{- end}}
-          {{- if (and .AzureEventHubs .AzureEventHubs.AuthUsingManagedIdentity) }}
+          {{- if .AzureEventHubsForKafka }}
           {
-            name: 'SPRING_CLOUD_AZURE_EVENTHUBS_CONNECTIONSTRING'
-            value: ''
```

Comment:
The empty string can be used to override the value specified in `application.yml`.